jovian/jovian_lab_ext/tsconfig.tsbuildinfo

```diff
@@ -0,0 +1,4519 @@
+{
+  "program": {
```

Comment:
please ignore this file in `.gitignore`

Reply:
Do I add **.gitignore** to **jovian/jovian_lab_ext/tsconfig.tsbuildinfo**?